app/controllers/users.js

```diff
@@ -18,3 +19,22 @@ exports.users = async (req, res, next) => {
     next(databaseError(DB_CONNECTION));
   }
 };
+
+exports.signIn = async (req, res, next) => {
+  try {
+    const mailFound = await findUserByEmail(req.body.mail);
+    const validatePass = await comparePass(req.body.pass, mailFound[0].pass);
+    if (!validatePass) next(compareError(COMPARE_PASS));
+    const payload = {
+      name: mailFound[0].name,
+      last_name: mailFound[0].last_name,
+      mail: req.body.mail
+    };
+    const token = generateToken(payload);
+    logger.info(`Successful login. Token: ${token}`);
```

Comment:
This could better be in an `interactor`. 

What is an interactor? It is an extra layer between the controllers and services/models. The idea is to organize the code to keep the controllers clean, without moving this business logic to services/models.